src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
```

Comment:
Writing everything in one line is not always a good idea, here it just reduces readability
```suggestion
  const {
    name,
    age,
    sex,
    ....
  } = person;
```